[2022-04-13 22:33:43,451] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:25+00:00 [queued]>
[2022-04-13 22:33:43,527] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:25+00:00 [queued]>
[2022-04-13 22:33:43,543] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:43,550] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:33:43,572] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:43,653] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-02-02 06:00:25+00:00
[2022-04-13 22:33:43,666] {standard_task_runner.py:52} INFO - Started process 10234 to run task
[2022-04-13 22:33:43,687] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-02-02T06:00:25+00:00', '--job-id', '1709', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmplnlgsbn_', '--error-file', '/tmp/tmpfs_8kai3']
[2022-04-13 22:33:43,695] {standard_task_runner.py:77} INFO - Job 1709: Subtask ingest
[2022-04-13 22:33:44,035] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:25+00:00 [running]> on host 13db72185839
[2022-04-13 22:33:44,378] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T06:00:25+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-02T06:00:25+00:00
[2022-04-13 22:33:44,386] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:33:44,389] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:33:44,418] {subprocess.py:85} INFO - Output:
[2022-04-13 22:33:44,435] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:33:44,437] {subprocess.py:89} INFO - dags
[2022-04-13 22:33:44,439] {subprocess.py:89} INFO - logs
[2022-04-13 22:33:44,440] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:33:44,441] {subprocess.py:89} INFO - plugins
[2022-04-13 22:33:44,442] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:33:44,442] {subprocess.py:89} INFO - scripts
[2022-04-13 22:33:44,443] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:33:44,444] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:33:44,549] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220202T060025, start_date=20220413T223343, end_date=20220413T223344
[2022-04-13 22:33:44,634] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:33:44,839] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
